### ISSUE TITLE
Couldn't get a file descriptor referring to the console

### ISSUE THREAD

**Issue by helio-frota** [nodeshift-starters/nodejs-circuit-breaker#32]
After run `./start-localhost.sh` :

```
Launching greeting service
Launching name service
To stop the servers, run "./shutdown-localhost.sh"
Couldn't get a file descriptor referring to the console
```

[updated]

This is related to the `open` command Mac specific

```
[hf@T440 ~]$ open
Couldn't get a file descriptor referring to the console
```

https://github.com/bucharest-gold/nodejs-circuit-breaker/blob/master/start-localhost.sh#L19

**Comment by helio-frota** [comment]
I'll send a PR now, tested here on fedora 27 and it is working.
But I need a feedback on Mac before merge.

**Comment by lance** [comment]
Should be fixed in #33
